docs/getting-started-auditor.md

```diff
@@ -0,0 +1,170 @@
+See [saclardl-samples](https://github.com/scalar-labs/scalardl-samples) repo for more details about how to use the containers.
+
+## Register each certificates of Ledger and Auditor
+
+As we explained,
+Ledger needs to register its certificate to Auditor, and Auditor needs to register its certificate to Ledger.
+This can be done by registering it as a client as follows:
+
+Ledger registers its certificate to Auditor
+```
+bin/register-cert --properties client.properties.ledger
+```
+
+Auditor registers its certificate to Ledger
+```
+bin/register-cert --properties client.properties.auditor
+```
+
+Please configure `client.properties.ledger` and `client.properties.auditor` properly.
+(This document assumes you know how to configure client.properties properly.)
+Note that `scalar.dl.client.cert_holder_id` has to be properly set to match with the one configured in Ledger and Auditor respectively.
+
+## Register your certificate
+
+You also need to update the properties file of a client before interacting with Scalar DL with Auditor.
+Please update the following entries:
+```
+[client.properties]
+scalar.dl.client.auditor.enabled=true
+scalar.dl.client.auditor.host=localhost
+```
+
+Then, you can register your certificate just like you have been doing usually (without Auditor).
+
+```
+client/bin/register-cert --properties client.properties
+```
+
+Note that this registers the certificate to both Ledger and Auditor.
+
+## Register your contracts
+
+For registering contracts, you can also do as usual.
+
+```
+client/bin/register-contract --properties client.properties --contract-id StateUpdater --contract-binary-name com.org1.contract.StateUpdater --contract-class-file build/classes/java/main/com/org1/contract/StateUpdater.class
+```
+
+Note that this registers the contract to both Ledger and Auditor.
+
+## Execute the contract
+
+Now you are ready to execute the contract with the following command as usual.
+
+```
+client/bin/execute-contract --properties client.properties --contract-id StateUpdater --contract-argument '{"asset_id":"some_asset", "state":3}'
+```
+
+Note that this triggers a little complex protocol between Ledger and Auditor to make them go to the same states without trusting each other.
+During the execution, it may detect inconsistencies between them if there is a tampering in one of the components.
+
+## Validate the states of Ledger and Auditor
+
+You can also always validate the states of Ledger and Auditor to see if they are consistent with the following command as usual:
+
+```
+client/bin/validate-ledger --properties client.properties --asset-id="some_asset"
+```
+
+By default, it checks the states of Ledger and Auditor non-linearizably; i.e., there might be cases where they look inconsistent temporarily.
+To make it linearizable, you can configure the following option and resister `ValidateLedger` contract as follows.
```

Comment:
I guess it's friendly if we provide an information about where the ValidateLedger contract is.

Reply:
Thank you! Fixed in 35168715abca80b7751c2b3949ba8445e0d88a7f.